dev-cpp/muparser/muparser-2.2.5.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="Mathematical expressions parser library"
+DESCRIPTION="Many applications require the parsing of mathematical \
+expressions. The main objective of this library is to provide a fast \
+and easy way of doing this. muParser is an extensible high performance \
+math expression parser library written in C++. It works by transforming \
+a mathematical expression into bytecode and precalculating constant parts \
+of the expression."
+HOMEPAGE="http://beltoforion.de/article.php?a=muparser"
+COPYRIGHT="2016 Ingo Berg"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/beltoforion/muparser/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0666ef55da72c3e356ca85b6a0084d56b05dd740c3c21d26d372085aa2c6e708"
+PATCHES="muparser-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	muparser$secondaryArchSuffix = $portVersion
+	lib:libmuparser$secondaryArchSuffix = $portVersion compat >= 2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	muparser${secondaryArchSuffix}_devel = $portVersion
+	devel:libmuparser$secondaryArchSuffix = $portVersion compat >= 2
+	"
+REQUIRES_devel="
+	muparser$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	cp `finddir B_SYSTEM_DATA_DIRECTORY`/libtool/config/config.guess build/autoconf/
+	cp `finddir B_SYSTEM_DATA_DIRECTORY`/libtool/config/config.sub build/autoconf/
```

Comment:
have to re-check for this, not sure, normally I check things out in Terminal first ...